core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -537,6 +559,22 @@ public void onClosed(Channel c, IOException cause) {
         Jenkins.getInstance().getQueue().scheduleMaintenance();
     }
 
+    /*package*/ static boolean isRelativePath(String path) {
+        if (path.startsWith("/"))
+            return false;
```

Comment:
Debatable whether you want to consider `\Windows\Temp\slave` absolute or relative. Technically it is drive-relative.